core/audits/dobetterweb/doctype.js

```diff
@@ -41,6 +43,7 @@ class Doctype extends Audit {
       failureTitle: str_(UIStrings.failureTitle),
       description: str_(UIStrings.description),
       requiredArtifacts: ['Doctype'],
+      __internalOptionalArtifacts: ['InspectorIssues'],
```

Comment:
If we're going to use this outside the experimental config, maybe drop the `__internal`?

Reply:
Not until FR does a better job re: scheduling. Also that's a bigger change :/